morango/sync/syncsession.py

```diff
@@ -376,42 +384,139 @@ def _pull_record_chunk(self, chunk_size, transfer_session):
         return self.session.get(self.urlresolve(api_urls.BUFFER), params=params)
 
 
+class SyncSignal(object):
```

Comment:
Yeah, this is very similar to Django's. In my experience, I generally try to avoid such things since the reason for dependency on Django isn't specifically for Signals. I think it helps keep this (standalone) package uncoupled and easier to maintain. If you feel strongly, I could use it. I think it would only save a few lines.

Reply:
Nope, this sounds good -- Django wouldn't handle the "grouping" stuff itself anyway.